mautic-inc/plugin-vtiger
# M3: Vtiger integration config form doesn't show up

JIRA: https://backlog.acquia.com/browse/MAUT-3263

This PR fixes Plugin's bundle inability to load `VtigerCrmIntegration` class.
As a result of this, config form is not displayed when we click on the vTiger plugin on the plugins page.

Steps to reproduce:
1. Make sure you are on developement branch of vtiger-plugin repository.
2. Make sure you are on `development` branch of Mautic cloud.
3. Open /s/plugins page, try to open vTiger plugin.
4. You will only see vTiger's logo and no config form.

Steps to test:
1. Checkout this branch (MAUT-3263).
2. Make sure you are on `development` branch of Mautic cloud.
3. Clear cache, redis cache, do `composer dumpautoload`.
4. Open /s/plugins page, try to open vTiger plugin.
5. You should be able to see the config form.
